alcs-frontend/src/app/shared/meeting-overview/meeting-overview.component.scss

```diff
@@ -229,26 +229,19 @@ mat-panel-description {
   }
 }
 
-:host::ng-deep .application-panel {
```

Comment:
no "host::ng-deep" anymore?

Reply:
Correct! Since its no longer a mat-expansion panel, I dont need to modify any deep css, just the surface layer.